docs/competition/virtual-competitions/README.md

```diff
@@ -0,0 +1,7 @@
+# Virtual Competitions
+
+Due to Covid-19 Student Robotics has held the competition for the SR2020 and SR2021 competition cycles online.
+
+[A simulator][simulator] has been developed that can be distributed to competitors so they can program their virtual robot. Teams then submit their code to us and we broadcast the matches.
```

Comment:
Worth noting here that it's a _new_ simulator, rather than existing and previously-distributed 2D one.

Reply:
Not sure I want to lock in the word 'new' here. I reckon specifying its a 3D simulator conveys the required information, anyone who used or was aware of the old 2D one will know its different.

Reply:
https://github.com/srobo/runbook/pull/98/commits/217c65a90f1deb0f338f01e7f91036deb82015e5